test/instrumentation/testdata/pkg/kubelet/metrics/metrics.go

```diff
@@ -160,10 +155,10 @@ var (
 	PLEGRelistInterval = metrics.NewHistogram(
 		&metrics.HistogramOpts{
 			Subsystem:      KubeletSubsystem,
-			Name:           PLEGRelistIntervalKey,
+			Name:           "test_histogram_metric",
```

Comment:
nit: Why change the name of this test metric? should it be changed back?

Reply:
I changed it to be clearer in the generated output.

Reply:
Ok I see. Just a little confused because other metrics above are all using the PLEG* naming.